unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/primitive/AbstractBooleanListTestCase.java

```diff
@@ -123,6 +123,15 @@ public void indexOf()
         Assert.assertEquals(-1L, emptyList.indexOf(false));
     }
 
+    @Test
+    public void boxed()
+    {
+        MutableBooleanList list = this.newWith(true, false, true);
+        MutableList<Boolean> actual = list.boxed();
+        BoxedMutableBooleanList expected = new BoxedMutableBooleanList(list);
```

Comment:
I think this should be a JDK `ArrayList` or EC `FastList` of `Boolean` to make sure the equals contract works correctly.

Reply:
Fixed. Thanks.